content/core-features/history.textile

```diff
@@ -17,3 +17,19 @@ h2(#persisted-history). Enabling persistent history
 Every message that is persisted to or retrieved from disk counts as an extra message towards your monthly quota. For example, for a channel that has persistence enabled, if a message is published, two messages will be deducted from your monthly quota. If the message is later retrieved from history, another message will be deducted from your monthly quota.
 
 To enable history on a channel, it is necessary to add a channel rule in the settings of your "application dashboard":https://faqs.ably.com/how-do-i-access-my-app-dashboard. See the "documentation on channel rules":https://faqs.ably.com/what-are-channel-rules-and-how-can-i-use-them-in-my-app for further information on what they are and how to configure them.
+
+h2(#ordering). Ordering of historical messages
+
+The order in which historical messages are returned by the History API is based on the message timestamp that was assigned by the channel in the region that the message was published in. This ordering is what Ably calls the canonical global order.
+
+It is important to note that this is not necessarily the order that messages were received by a realtime client, which depends on which region the client is in. 
+
+We preserve ordering for a specific publisher on a specific channel but, for example, if two publishers in regions A and B publish "message 1" and "message 2" simultaneously, then it is very possible that a subscriber in region A will receive "message 1" before "message 2", but that a subscriber in region B will receive "message 2" before "message 1".
+
+There are instances where messages will not be in canonical global order:
+
+- Recent messages (less than two minutes old) are retrieved from live ephemeral storage and are still ordered by region. They only appear in the canonical global order if you enable "message persistence":/realtime/history#persisted-history (with a mechanism that prevents duplication and missing messages).
```

Comment:
This reads as the developer needing to enable message persistence **_and write_** a mechanism to prevent duplication and missed messages. I'm not sure if we need to include that part as it should be covered in the persistence docs (hopefully).

>(with a mechanism that prevents duplication and missing messages)

Reply:
That's a really good point. Changed for clarity.